scripts/generate-locale-messages.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const mkdirp = require('mkdirp');
+const languages = require('../languages.json');
+
+const LANG_DIR = './translations/';
+const MSGS_DIR = './locale/';
+
+const locales = Object.keys(languages);
+let messages = locales.reduce((collection, lang) => {
```

Comment:
Would you please add a file description to this script as well? Not immediately clear what this does.